lib/pure/os.nim

```diff
@@ -965,6 +965,37 @@ proc quoteShellWindows*(s: string): string {.noSideEffect, rtl, extern: "nosp$1"
   if needQuote:
     result.add("\"")
 
+when defined(windows):
+  proc quoteShellWindowsForSh*(s: string): string {.noSideEffect, rtl, extern: "nosp$1".} =
```

Comment:
Remove this please

Reply:
I'm removing it to unblock this PR but it is useful in case user wants to use non-standard sh or bash instead of standard (but fraught with limitations) cmd.exe